lib/std/os.zig

```diff
@@ -4978,25 +4979,26 @@ pub const CopyFileRangeError = error{
 /// Other systems fall back to calling `pread` / `pwrite`.
 ///
 /// Maximum offsets on Linux are `math.maxInt(i64)`.
```

Comment:
This doc-comment is now on the wrong thing I think?

Reply:
Duh, I'll fix that.